AUDIT_COMPLET_GOOGLE_CALENDAR_ONLY.md

```
@@ -0,0 +1,668 @@
+RESEND_API_KEY=re_Crbni8Gw_2Jb32KcyR4gLdkGs8umzGrHd
+
+# URL de l'application
+APP_URL=https://planning-doriansarry47-creators-projects.vercel.app
+
+# Environnement
+NODE_ENV=production
+```
+
+### Configuration sur Vercel
+
+1. Aller sur https://vercel.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)